drfx/settings.py

```diff
@@ -343,3 +328,34 @@
     from .settings_local import *  # noqa
 except ImportError:
     pass
+
+# Constance config - configure settings for the app that are editable in django admin
+CONSTANCE_CONFIG = {
+    "ASSOCIATION_RULES_URL": (
+        "https://tampere.hacklab.fi/pages/yhdistyksen-s%C3%A4%C3%A4nn%C3%B6t/",
+        "Link to the rules of the association",
+        str,
+    ),
+    "MEMBERS_GUIDE_URL": (
+        "https://wiki.tampere.hacklab.fi/member_s_guide",
+        "Link to the guide for new members",
+        str,
+    ),
+    "SITENAME": ("Hacklab Mulysa", "Name of the hacklab", str),
+    "SITE_URL": ("https://mulysa.tld", "URL of this site", str),
+    "PRIVACY_POLICY_URL": (
+        "https://example.com/privacy_policy.html",
+        "Link to privacy policy",
+        str,
+    ),
+    # Receipt functionality configuration
+    "RECEIPT_NAME": ("Mulysa ry", "Name of the association to show on receipts", str),
+    "RECEIPT_REGID": ("1234567-8", "ID of the association to show on receipts", str),
+    "RECEIPT_ADDRESS": (
+        "Street 12, Somewhere Finland",
+        "Address to show on receipts",
+        str,
+    ),
+    "ACCOUNT_IBAN": ("FI12 3456 789", "IBAN of the association's bank account"),
```

Comment:
I'm not sure if I want this here.

Even tho I trust our users this is the one field that can cause quite a lot of grief if someone decides to  change the target IBAN.

It might take a few weeks for anyone to really notice that all payments are going to bahamas.

Reply:
@tswfi We gotta trust our (admin) users, otherwise `rm -rf mulysa` is the only viable option.

Reply:
Then mulysa needs more levels of users, aka more use of the already existing staff tickbox or something alike.

Reply:
Until that happens I would prefer keeping this in a file where I know that no sql injection or malicious user can change it.

Reply:
I will make some changes so that you can make it so through settings.local.py

Reply:
I added ConfigAccessor, which is a class that is now used to access constance config and settlings all in one place. This means that the application code doesn't care whether a variable comes from constance or `settings.py`/`settings_local.py`. Simply remove variables from `CONSTANCE_CONFIG` to use a variable with the same name from `settings.py`/`settings_local.py` instead!

Reply:
So in your use case you can safely copy over the `CONSTANCE_CONFIG` from settings.py into settings_local.py and just remove the IBAN field. 

Note that when a variable is defined in both settings.py and constance, constance takes priority.